colcon-build-and-test/action.yaml

```diff
@@ -72,7 +72,7 @@ runs:
         colcon test --event-handlers console_cohesion+ \
           --packages-above ${{ inputs.target-packages }} \
           --return-code-on-test-failure
-        colcon lcov-result --packages-above ${{ inputs.target-packages }}
+        (find build -name coverage.info | grep .) && colcon lcov-result --packages-above ${{ inputs.target-packages }}
         colcon coveragepy-result --packages-above ${{ inputs.target-packages }}
```

Comment:
Don't you need the same thing for `coveragepy-result`?

Reply:
We don't need to add the check statement because it doesn't return non-zero code.

```
❯ colcon coveragepy-result 
...
Summary: 165 packages finished [0.44s]
[0.630s] WARNING:colcon.colcon_coveragepy_result.verb.coveragepy_result:No coverage files found

❯ echo $status
0
```

Reply:
I see, but I'm concerned that sometimes it will change.

Reply:
I modified condition. a975c35

Reply:
Does `coveragepy-result` use `coverage.info`, too?

Reply:
No, `coveragepy-result` uses `.coverage`.
https://github.com/colcon/colcon-coveragepy-result/blob/master/colcon_coveragepy_result/verb/coveragepy_result.py#L107

So, I changed condition(Thanks for your comment!). 83613f5

Also, I evaluated about rclpy coverage option and I noticed that we need additional options. fd25f16
The options come from here:
https://github.com/colcon/colcon-mixin-repository/blob/master/coverage.mixin

Reply:
Why you don't use `colcon-mixin` here? :thinking:

Reply:
It could have been done either way, but I think it's better to write the options directly, so we don't have to rely on `colcon-mixin`.

Reply:
No, I don't think so. I guess `colcon-mixin` was created so that all developers don't have to write the same options every time.

Reply:
It's fine. That change is acceptable to me.

Reply:
5859cb7